libraries/botbuilder-ai/botbuilder/ai/luis/luis_recognizer.py

```diff
@@ -362,3 +370,11 @@ def fill_luis_event_properties(
         )
 
         await turn_context.send_activity(trace_activity)
+
+    def _merge_options(
+        self,
+        userDefinedOptions
+    ) -> LuisPredictionOptions:
+        mergedOptions = LuisPredictionOptions()
```

Comment:
If possible, consider changing this to snake_case :)

Reply:
agree...updated to snake case